src/main/java/Data/ServerDatabaseCommunicator.java

```diff
@@ -0,0 +1,453 @@
+package Data;
+
+import Model.Group;
+import Model.Notification;
+import Model.Response;
+import org.apache.wink.json4j.JSONArray;
+import org.apache.wink.json4j.JSONException;
+import org.apache.wink.json4j.JSONObject;
+import org.opendatakit.aggregate.odktables.rest.entity.Column;
+import org.opendatakit.aggregate.odktables.rest.entity.Row;
+import org.opendatakit.sync.client.SyncClient;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.*;
+
+
+public class ServerDatabaseCommunicator {
```

Comment:
@DreamyPhobic @codeknight3 Can we make this class a singleton class? What are your thoughts?